### PR TITLE
Feature/show connected apis card

### LINKED ISSUE #134: Show nr of connected APIs in organization card
The below left corner of organization card should display and indicator the number of connected APIs for each organization.

If possible,  use the same implementation and style to show the number of connected APIs on the organization profile. Size of indicator could  scaled up for the profile page. 

# User story
> As an organization manager, I want to show the number of connected APIs per organization
so that it makes my organization more appealing to developers.

# Definition of done
* [ ] There is a number and an icon representing the amount of connected APIs in the below left corner of organization card.
* [ ] If possible, there is a number and an icon representing the amount of connected APIs on organization profile page.

# Wireframe

Organization Card
organizationcard

In Organization Profile
The no. of APIs connected to the organization appears on the header of organization profile.
Between organization URL and Contact information.
In this way, user can always see no. of APIs connected if s/he switches between tabs (in the case tab design is implemented)
connectedapis

### PR DESCRIPTION
**Pull request by jykae**
Closes #134

### CODE REVIEW

**Comment by jykae** [comment]
Waiting PR #201 to get "organization.apisCount" collection helper.

**Comment by marla-singer** [comment]
@jykae 201 PR Is merged

File: organizations/client/catalog/grid/grid.html [hunk 1]
```diff
@@ -20,6 +20,11 @@
             {{/ if }}
           </div>
           <ul class="organization-card-stats list-inline">
+            <li class="pull-left">
+              <span class="organization-card-apis-count">
+                {{ organization.apisCount }} {{_ "organizationCatalogGrid_apisText" }}
```
Comment (brylie) [inline organizations/client/catalog/grid/grid.html]: This i18n token should follow the form:

> templateName_pageElement_additionalText

It currently omits the page element, including only the 'additional text' (`apisText`).

Consider instead:

> organizationCatalogGrid_organizationCardApisCount_text

**Comment by brylie** [comment]
Good work. Suggested small change to i18n key. Can merge quickly when change is made.

**Comment by jykae** [comment]
@brylie i18n keys changed based on contributing guidelines

**Pull request merged**